src/mixer/basetrackplayer.h

```diff
@@ -123,6 +123,8 @@ class BaseTrackPlayerImpl : public BaseTrackPlayer {
     bool m_replaygainPending;
     EngineChannel* m_pChannelToCloneFrom;
 
+    PerformanceTimer m_ejectTimer;
```

Comment:
IMO `PerformanceTimer` is a little overkill, but we don't have anything better in mixxx right now (except possibly implementing a custom clock based on `std::chrono`). Unimportant.